src/routes/+layout.svelte

```diff
@@ -5,8 +5,15 @@
 	import '../app.css';
 	import Header from './_components/Header.svelte';
 	import { configureHighcharts } from '$lib/charts/baseChart';
+	import { onMount } from 'svelte';
 
 	let { children, data } = $props();
+
+	// Add 'started' class to body when app is mounted to indicate hydration is complete
```

Comment:
annoying check,, used in playwright tests to ensure app is mounted. its in root layout as that wraps whole app